qutebrowser/mainwindow/prompt.py

```diff
@@ -630,6 +630,20 @@ def __init__(self, question, parent=None):
 
         self._to_complete = ''
 
+    def _directories_hide_show_model(self, path):
+        """Get rid of non-matching directories."""
+        try:
+            num_rows = self._file_model.rowCount(self._root_index)
+            for row in range(num_rows):
+                index = self._file_model.index(row, 0, self._root_index)
+                hidden = self._to_complete not in index.data()
+                if index.data() == '..':
+                    hidden = False
+                self._file_view.setRowHidden(index.row(), index.parent(), hidden)
+        except FileNotFoundError:
```

Comment:
Now that we're iterating over the model rather than re-fetching the data from disk, I don't think there's a way this can ever happen?

Reply:
Yep that is correct, I should remove that.